kind-image/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG GO_VERSION
+
+FROM lachlanevenson/k8s-kubectl:v1.17.2 as kubectl
+
+FROM golang:${GO_VERSION}-alpine3.11 as golang
```

Comment:
I'm wondering if we could remove Go from this image now, as we are not _building_ Kind from source anymore.

Reply:
Good question.  Although we still need a minimal golang installation to support the current iteration of the e2e script (for example, see https://github.com/brigadecore/brigade/blob/master/e2e/run.sh#L80), we can definitely remove some of the extra crud that had existed to run full golang builds...